Documentation/data-collection.md

```diff
@@ -0,0 +1,60 @@
+# OpenShift 4 Data Collection
+
+Red Hat values our customer experience and privacy. It is important to us that our customers understand exactly what we are sending back to engineering and why. During the developer preview or beta release of our software, we want to be able to make changes to our designs and coding practices in real-time based on customer environments. The faster the feedback loop during these development stages the better.
```

Comment:
I am not sure this is true only for beta and developer releases, we always want to make sure this works? Or will we stop sending this data at some point?

Reply:
I used the existing docs in the telemeter repo for this, so this has been signed off previously, but pinging @sichvoge for extra review.